### DISCUSSION TITLE
I suddenly can't access to GitHub Copilot

### DISCUSSION THREAD

**Discussion by yozurukt** [community/community#112449] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

![image](https://github.com/community/community/assets/59713727/134c9dc5-4cee-414a-8947-b80eeeb9109f)
![image](https://github.com/community/community/assets/59713727/d416150b-e93e-4bfa-9685-2e765aed647f)
My github student pack hasn't expired yet and it shows that I still seem to be able to use github copilot for free.
But the truth is that I am prompted with the following in all jetbrains ide
![2d53136affe221dfe75c3393aba212bb](https://github.com/community/community/assets/59713727/39e3c19c-6e22-4e34-93ca-6e652f4b6bf8)
It worked fine for me for the first couple days and I didn't make any changes.
What is concerning is that in the first picture it shows that my next PAYMENT is today. Could this be the cause of the problem? But either way I don't think it makes sense. I need to get help on this.

**Comment by AidenJamesYt** [comment, marked as answer]
i would open a ticket

here is how you can do so: https://docs.github.com/en/support/contacting-github-support/creating-a-support-ticket

**Comment by Traspa1112** [comment]
Hi, were you able to fix that? I'm scared of being charged. Thanks!
